.github/workflows/release.yml

```
@@ -0,0 +1,92 @@
+name: Create GitHub Release
```

```suggestion
name: Build and publish AppImages
```

Since this is no longer exclusive to releases.

---

Done.